rest-api/src/main/java/gov/cms/qpp/conversion/api/model/Metadata.java

```diff
@@ -494,6 +491,18 @@ public boolean equals(Object o) {
 		return equals;
 	}
 
+	public static class InstantConverter implements DynamoDBTypeConverter<String, Instant> {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Document this public class by adding an explicit description. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUndocumentedApi)